pkg/utils/utils.go

```diff
@@ -219,6 +219,14 @@ func IsCatchAllIPEntry(ipAddr string) bool {
 	return false
 }
 
+func IsNodeIP(nodeIP string, ipCidr string) bool {
```

Comment:
Is there any special handle needed for IPv6 compact format?

Reply:
net.Parse* functions should take care of that